packages/create-qwik/create-interactive.ts

```diff
@@ -71,7 +71,13 @@ export async function runCreateInteractiveCli() {
     }
 
     if (existingOutDirAnswer === 'replace') {
-      removeExistingOutDirPromise = fs.promises.rm(outDir, { recursive: true });
+      fs.promises
+        .readdir(outDir)
+        .then((filePaths) =>
+          filePaths.forEach((pathToFile) =>
+            fs.promises.rm(path.join(outDir, pathToFile), { recursive: true })
```

Comment:
thx a lot for the adjustments 🙏 will give it a spin asap 👍 

regarding the code changes: i guess you could write it just like this:
```suggestion
            fs.promises.rm(join(outDir, pathToFile), { recursive: true })
```
since `join` is already a named import 🙏

Reply:
Right, I will commit and push it now

Reply:
Done